packages/flutter_tools/lib/executable.dart

```diff
@@ -135,36 +139,80 @@ Future<void> main(List<String> args) async {
        WebRunnerFactory: () => DwdsWebRunnerFactory(),
        // The mustache dependency is different in google3
        TemplateRenderer: () => const MustacheTemplateRenderer(),
-       if (daemon)
-        Logger: () => NotifyingLogger(
```

Comment:
dang, didn't know we had all these

Reply:
![PB9d7](https://user-images.githubusercontent.com/8975114/87119152-04a07780-c232-11ea-922a-d5aea8784f2a.jpg)